docs/quickstart/quickstart_chains/ethereum-uniswap.md

```diff
@@ -0,0 +1,1111 @@
+# Ethereum Quick Start - Uniswap (Medium)
+
+## Goals
+
+Within the domain of DeFi, the objective of indexing Uniswap entities is both crucial and versatile. Essentially, indexing serves as the foundation for transparency and ease of use in the Uniswap system. By systematically organizing tokens, liquidity pools, transactions, and other essential information, indexing provides users with a quick and efficient means to search, find, and analyze data.
+
+The objective of this article is to offer a detailed, step-by-step guide on setting up a SubQuery indexer for Uniswap data indexing. We will comprehensively cover the necessary configurations and delve into the intricacies of the underlying logic.
+
+Previously, in the [1. Create a New Project](../quickstart.md) section, you must have noted [3 key files](../quickstart.md#_3-make-changes-to-your-project). Let's begin updating them one by one.
```

Comment:
@bgodlin 

I like all the changes. I have just one question related with the guide dependencies, previous steps reader needs to take to follow it. Shall we mention which project should this person initialise via SQ CLI?

Reply:
Added as part of https://github.com/subquery/documentation/pull/415/commits/b2000991d87bd155b782b0eb540b2c5b1e6ec632